pkg/plugin/trans.go

```diff
@@ -73,9 +83,13 @@ func (s verdictSet) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 // any disconnect, which could lead to unintentional over-committing of resources
 // from the Buffer values if too many agents request upscaling on the first
 // request to the scheduler.
-func (r resourceTransitioner[T]) handleLastPermit(lastPermit T) (verdict string) {
+func (r resourceTransitioner[T]) handleLastPermit(lastPermit T, overcommit *float64) (verdict string) {
```

Comment:
I think we need to, yeah - the current implementation returns the incorrectly scaled values from `(resourceTransitioner[T]).handleRequested(...)`, and in order to fix without rounding issues, we'd need to operate as if the node has more capacity, rather than as if the VM uses less.